src/ngraph/frontend/onnx_import/op/pad.cpp

```diff
@@ -18,19 +18,46 @@
 
 #include "default_opset.hpp"
 #include "exceptions.hpp"
+#include "ngraph/builder/split.hpp"
 #include "ngraph/coordinate_diff.hpp"
 #include "ngraph/op/constant.hpp"
+#include "ngraph/op/convert.hpp"
 #include "ngraph/op/pad.hpp"
 #include "ngraph/shape.hpp"
 #include "pad.hpp"
 #include "utils/convpool.hpp"
 
+using namespace std;
+
 namespace ngraph
 {
     namespace onnx_import
     {
         namespace op
         {
+            ngraph::op::PadMode setPadMode(string mode)
```

Comment:
Please make this function `static` in global namespace or put it inside _anonymous namespace_ - that's the right place for file specific helper functions and this is a good practice.